data/formats-data.js

```diff
@@ -6360,4 +6360,94 @@ exports.BattleFormatsData = {
 		isNonstandard: true,
 		tier: "CAP",
 	},
+ +	brycenman: {
```

Comment:
It took me a long time to figure out why there are two plus signs here, and it turns out the plus signs are plus signs you accidentally inserted.